openxc-ios-framework/Command.swift

```diff
@@ -86,7 +86,7 @@ open class Command: NSObject {
     fileprivate var managerDebug : Bool = false
 
     // data buffer for storing vehicle messages to send to BTLE
-    fileprivate var BLETxDataBuffer: NSMutableArray! = NSMutableArray()
+   // fileprivate var BLETxDataBuffer: NSMutableArray! = NSMutableArray()
```

Comment:
why is this commented? we need this as NSMutableArray()

Reply:
This is moved to vechile manger class .Reason at the end we are  assigning the value to variable  in vehicle manager class.